src/core/server/graph/schema/schema.graphql

```diff
@@ -1269,6 +1269,38 @@ type StoryMessageBox {
   content: String
 }
 
+enum GIPHY_RATING {
+  G
+  PG
+  PG13
+  R
+}
+
+################################################################################
+## Embed Links
+################################################################################
+
+type EmbedConfiguration {
+  """
+  twitter when true will enable twitter link embeds in the stream.
+  """
+  twitter: Boolean!
+
+  """
+  youtube when true will enable youtube link embeds in the stream.
+  """
+  youtube: Boolean!
+
+  """
+  giphy
+  """
+  giphy: Boolean!
+
+  giphyMaxRating: GIPHY_RATING!
```

Comment:
We can store this as a string for now (we don't typically create enums for external properties).